htdocs/takepos/index.php

```diff
@@ -802,7 +806,7 @@ function WeighingScale(){
 	Refresh();
 	<?php
 	//IF NO TERMINAL SELECTED
-	if ($_SESSION["takeposterminal"] == "") {
+	if (isset($_SESSION["takeposterminal"]) && $_SESSION["takeposterminal"] == "") {
```

Comment:
if $_SESSION["takeposterminal"] is not defined the test 
`if ($_SESSION["takeposterminal"] == "") {`
will be true. To keep same behavior, you replace with
`if (!isset($_SESSION["takeposterminal"]) || $_SESSION["takeposterminal"] == "") {`